lib/mix/lib/releases/models/profile.ex

```diff
@@ -15,7 +15,7 @@ defmodule Mix.Releases.Profile do
     include_src: nil, # boolean
     include_system_libs: nil, # boolean | "path/to/libs"
     strip_debug_info: nil, # boolean
-    plugins: nil, # list of module names
+    plugins: [], # list of module names
```

Comment:
We need to handle this [here](https://github.com/bitwalker/distillery/blob/master/lib/mix/lib/releases/assembler.ex#L64), since changing this to `[]` means that whether you have plugins defined in the environment or not, it will always override whatever is defined in the release. We probably just need to add a case for `[]` and treat it like we do `nil` (e.g. ignore it).